health_check/views.py

```diff
@@ -77,36 +78,12 @@ def __lt__(self, other):
         return self.weight.__lt__(other.weight)
 
 
-class MainView(CheckMixin, TemplateView):
+class MainView(CheckMixin, generics.RetrieveAPIView):
     template_name = 'health_check/index.html'
+    renderer_classes = [TemplateHTMLRenderer, JSONRenderer]
+    serializer_class = HealthSerializer
 
-    @never_cache
```

Comment:
Are DRF responses not cached in any circumstances by default?

Reply:
My mistake, this line was added back in yesterdays commit